src/material/datepicker/month-view.html

```diff
@@ -1,6 +1,10 @@
 <table class="mat-calendar-table" role="presentation">
   <thead class="mat-calendar-table-header">
-    <tr><th *ngFor="let day of _weekdays" [attr.aria-label]="day.long">{{day.narrow}}</th></tr>
+    <tr>
+      <th *ngFor="let day of _weekdays" [attr.aria-label]="day.long">
```

Comment:
Note: I wasn't sure whether we should keep the `aria-label` here, considering that the same info is already on the `abbr`'s title. What do you think @jelbourn?

Reply:
In both VoiceOver and ChromeVox it behaves the same way with or without the `aria-label`. Can you try NVDA?

Reply:
I'm having some issues with my Windows computer, but I can try it out next week.

Reply:
@jelbourn I finally got around to testing it in NVDA and it behaves the same way with or without the `aria-label`: either way it doesn't read out the full name of the day. How should we proceed here? These changes don't degrade the experience, but they don't seem to be making it any better for NVDA users either.